lib/vagrant-libvirt/action/create_network_interfaces.rb

```diff
@@ -193,7 +193,7 @@ def call(env)
               # It's used for provisioning and it has to be available during provisioning,
               # ifdown command is not acceptable here.
               next if slot_number.zero?
-              next if options[:auto_config] === false
+              next if !options[:auto_config]
```

Comment:
With the work to push this up with separate changes, it's been discovered that this causes a failure for the network configuration because it previously was been used to only skip if `auto_config` was explicitly set to false, whereas this changes it to skip unless it's set to `true` or a `truthy` value.

Consequently, this piece has been removed from what was merged.

Was this just an accidental change or was there something this fixed that needs to be looked at closer?

Reply:
The main point of changes in that module was to allow tcp tunnels auto_config, so I think it's ok to set to explicit false value check in case it fails on some cases. The more important change is below this line where direct check on tunnels was removed.
Regards.